.github/workflows/build-and-deploy.yml

```diff
@@ -72,6 +76,18 @@ jobs:
           npm install
           cd ../../
           
+      - name: install for Journeys/app
+        run: |
+          cd journeys/app
+          npm install
+          cd ../../
+          
+      - name: npm install for Journey/scripts
+        run: |
+          cd journeys/scripts
+          npm install
```

Comment:
I think this could be to late here as you are already calling the script in line 65